src/element-stub.js

```diff
@@ -27,6 +27,11 @@ export class ElementStub extends BaseElement {
   }
 
   /** @override */
+  getPriority() {
+    throw new Error('Cannot get priority of stubbed element');
```

Comment:
This is fine. Both do very similar things and both we want to show up in our production error reporting, because they are not expected to actually happen.